File: backend/src/main/java/nus/iss/backend/dao/MedicationResponse.java
package nus.iss.backend.dao;

import lombok.Getter;
import lombok.Setter;

import java.util.UUID;

@Getter
@Setter
public class MedicationResponse {
    private UUID id;
    private String medicationName;
    private String intakeQuantity;
    private int frequency;
    private String timing;
    private String instructions;
    private String note;
    private Boolean isActive;
}

File: android/app/src/main/java/com/example/medimind/service/MedicationResponse.kt
package com.example.medimind.service

data class MedicationResponse(
    val id: String,
    val medicationName: String,
    val intakeQuantity: String,
    val frequency: Int,
    val timing: String?,
    val instructions: String?,
    val notes:String?,
    val active: Boolean,
    val scheduleIds: List<String>
)
